misaligned.js

```diff
@@ -11,6 +11,9 @@ function print_color_map() {
     return majorColors.length * minorColors.length;
 }
 
+// Run the function
 result = print_color_map();
-expect(result).equals(25);
+expect(result).equals(25); 
+expect(`0 | White | Blue`).to.equal(" 0  | White     | Blue"); // Misaligned output (fails)

```

Comment:
This will fail even after you have fixed the code, right? Idea is that the tests should pass when the functionality is correct.
You had asserted the result in your previous assignment... you can do similar here?